### PR TITLE
a few small changes

### PR DESCRIPTION
**Pull request by maxcroasdale**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,16 @@
   "license": "LGPL-3.0",
   "dependencies": {
     "truffle": "^5.1.23",
+    "truffle-keystore-provider": "github:shirikatsu/truffle-keystore-provider",
```
Comment (cmjc) [inline package.json:26]: why does this say shirikatsu in the path?

**Comment by Andreadinenno** [comment]
Hi Max, thanks for the typos fix! I'm gonna close this cause we don't merge to master until it's a release. Since you pushed to develop then we're fine! @cmjc

File: docs/IERC1820Registry.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# [🔗](/interfaces/IERC1820Registry.sol#L3) IERC1820Registry
+**Author** _Jordi Baylina and Jacques Dafflon_
```
Comment (cmjc) [inline docs/IERC1820Registry.md:2]: What was the source repo for this? Should we not acknowledge that as well as authors? @maxcroasdale @Andreadinenno  what is the engineering protocol and technical writing best practice here, please?

File: docs/IERC1820Implementer.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# [🔗](/interfaces/IERC1820Implementer.sol#L3) IERC1820Implementer
+**Author** _Jordi Baylina and Jacques Dafflon_
```
Comment (cmjc) [inline docs/IERC1820Implementer.md:2]: What was the source repo for this? Should we not acknowledge that as well as authors? @maxcroasdale @Andreadinenno what is the engineering protocol and technical writing best practice here, please?

File: interfaces/IERC1820Registry.sol [hunk 4]
```diff
@@ -0,0 +1,85 @@
+pragma solidity 0.5.3;
+
+/**
+* @title ERC1820 Pseudo-introspection Registry Contract
+* @author Jordi Baylina and Jacques Dafflon
```
Comment (cmjc) [inline interfaces/IERC1820Registry.sol:5]: Can we make this metadata consistent across all the `.sol` please? See comments on @author and what this metadata should be.

**Review by cmjc — CHANGES_REQUESTED** [review]
Just questions on the metadata and author acknowledgement.

As a check

File: package.json (hunk 1 above)
Comment (Andreadinenno) [reply package.json:26]: Chris forked the repo and add fixes to be able to deploy over https.

File: docs/IERC1820Implementer.md (hunk 3 above)
Comment (Andreadinenno) [reply docs/IERC1820Implementer.md:2]: Technically these natspecs are the one of the original erc1820 so they're the authors.

File: package.json (hunk 1 above)
Comment (cmjc) [reply package.json:26]: Yes, but surely this shouldn't be in the package.json in the release?

File: docs/IERC1820Implementer.md (hunk 3 above)
Comment (cmjc) [reply docs/IERC1820Implementer.md:2]: So source repo is https://github.com/0xjac/ERC1820/blob/master/contracts/ .

**Pull request closed without merging**